docker/resource_docker_container_test.go

```diff
@@ -610,6 +610,98 @@ func TestAccDockerContainer_upload(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccContainerRunning("docker_container.foo", &c),
 					testCheck,
+					resource.TestCheckResourceAttr("docker_container.foo", "name", "tf-test"),
+					resource.TestCheckResourceAttr("docker_container.foo", "upload.#", "1"),
+					// NOTE mavogel: current the terraform-plugin-sdk it's likely that
```

Comment:
Can this resource support import? If so, we can add an import test step that can check all these values instead of manually figuring out the Set hashcode.

Reply:
A PR for imports #196 is also in progress and I'll update then test then. Thanks for the hint